pages/_app.tsx

```diff
@@ -14,6 +14,7 @@ export default function App({ Component, pageProps }: AppProps) {
     <>
       <Head>
         <title>Ny i Abakus</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
Dunno where it's best to keep the meta tags, but why move one and not all of them?

Reply:
It gave an error specifically for the `viewport meta tag`, so apparently it may <i>..lead to unexpected results since it cannot be deduped. The viewport tag should be handled by next/head in pages/_app.js</i> (quoted from ![next docs](https://nextjs.org/docs/messages/no-document-viewport-meta)). It also said the solution was simply to move it 😄